### PR TITLE
Invites management

### PR DESCRIPTION
**Pull request by Morpheo1**

### CODE REVIEW

**Review by MrZaiko — APPROVED** [review]
Perfecto

File: app/src/main/java/io/github/polysmee/invites/InvitesManagementActivity.java [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        setListenerUserAppointments();
+    }
+
+    /**
+     * Sets up the invitation's view to display all the correct informations
+     *
+     * @param appointment the appointment's whose description is created
+     * @param InviteEntry the invitation's view
+     */
+    protected void makeInviteEntry(CalendarAppointmentInfo appointment, View InviteEntry) {
```
Comment (Fermeli) [inline app/src/main/java/io/github/polysmee/invites/InvitesManagementActivity.java:66]: Consider adding defensive programming, @NonNull for exemple

**Review by Fermeli — APPROVED** [review]
All good

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -7,22 +7,31 @@
...
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:largeHeap="true"
         android:requestLegacyExternalStorage="true"
+        android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Polysmee"
         android:usesCleartextTraffic="true">
+        <activity android:name=".invites.InvitesManagementActivity">
```
Comment (MRandl) [inline app/src/main/AndroidManifest.xml:28]: is it normal if you added a main activity ?

**Review by MRandl — APPROVED** [review]
maybe fix that thing where there's an extra main but the rest is all good

**Pull request merged**